### PR TITLE
control-service: Expose LOGGING_FORMAT through helm chart

### PR DESCRIPTION
**Pull request by gageorgiev**
Currently, the logging format of the control service as set by the LOGGING_FORMAT variable cannot be set through the helm chart. This PR corrects that by
exposing it.

Signed-off-by: Gabriel Georgiev <gageorgiev@vmware.com>

### CODE REVIEW

**Comment by tozka** [comment]
Why do we need it exposed?

**Comment by gageorgiev** [comment]
> Why do we need it exposed?

To configure VDP CS logs to json, so they can be parsed in Log Insight Cloud.

File: projects/control-service/projects/pipelines_control_service/src/main/resources/application-prod.properties [hunk 1]
```diff
@@ -131,4 +131,4 @@ datajobs.monitoring.sync.initial.delay=${DEFAULT_MONITORING_INITIAL_DELAY_SYNC_I
 #datajobs.deployment.builder.extraArgs=${DATAJOBS_DEPLOYMENT_BUILDER_EXTRAARGS}
 
 # Enables JSON-formatted logging for all logs emitted by the Control Service.
-# LOGGING_FORMAT=JSON
+LOGGING_FORMAT=${CONTROL_SERVICE_LOGGING_FORMAT:""}
```
Comment (tozka) [inline projects/control-service/projects/pipelines_control_service/src/main/resources/application-prod.properties:134]: pipelinesControlService is redundant. For other control service options, we do not set a prefix. As it's a pipelines control service chart it's implied. For example, we have "notificationOwnerName" and not "controlServiceOwnerName" . "loggingFormat" or "serviceLoggingFormat" is enough

**Review by tozka — APPROVED** [review]

File: projects/control-service/projects/pipelines_control_service/src/main/resources/application-prod.properties (hunk 1 above)
Comment (gageorgiev) [reply projects/control-service/projects/pipelines_control_service/src/main/resources/application-prod.properties:134]: gotcha, I'll change it

**Review by murphp15 — APPROVED** [review]

**Pull request merged**